tests/Aspire.Workload.Tests/Aspire.Workload.Tests.csproj

```
@@ -0,0 +1,56 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+
+    <RunTestsOnHelix>true</RunTestsOnHelix>
+    <TestUsingWorkloads>true</TestUsingWorkloads>
+    <InstallWorkloadForTesting>true</InstallWorkloadForTesting>
+
+    <XunitRunnerJson>xunit.runner.json</XunitRunnerJson>
+    <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
+    <TestArchiveTestsDir>$(TestArchiveTestsDirForWorkloadTests)</TestArchiveTestsDir>
+
+    <InstallChromeForPlaywright Condition="'$(InstallChromeForPlaywright)' == '' and '$(CODESPACES)' == 'true'">true</InstallChromeForPlaywright>
+    <InstallChromeForPlaywright Condition="'$(InstallChromeForPlaywright)' == '' and '$(ArchiveTests)' == 'true' and '$(ContinuousIntegrationBuild)' == 'true'">true</InstallChromeForPlaywright>
+
+    <ExtractTestClassNamesForHelix Condition="'$(ContinuousIntegrationBuild)' == 'true' or '$(ArchiveTests)' == 'true'">true</ExtractTestClassNamesForHelix>
+  </PropertyGroup>
+
+  <Import Project="..\Shared\provisioning.targets" />
+  <Import Project="..\Shared\WorkloadTesting\Aspire.Shared.WorkloadTesting.targets" />
+
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)src\Aspire.Hosting.Redis\RedisContainerImageTags.cs" />
+
+    <None Include="data\**\*" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="$(RepoRoot)eng\helix\runtests.*" CopyToOutputDirectory="PreserveNewest" />
+
+    <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
+    <PackageReference Include="Microsoft.Extensions.Http.Resilience" />
+  </ItemGroup>
+
+  <!-- Used for running one helix job per test class -->
+  <Target Name="_ExtractTestClassNames"
+          Condition="'$(ExtractTestClassNamesForHelix)' == 'true'"
+          BeforeTargets="ZipTestArchive">
+
+    <Exec Command="&quot;$(DotNetTool)&quot; test --no-build -c $(Configuration) -s $(RunSettingsFilePath) -t --nologo -v:q -p:VsTestUseMSBuildOutput=false" ConsoleToMSBuild="true">
```

```suggestion
    <Exec Command="&quot;$(DotNetTool)&quot; test --no-build -c $(Configuration) -s $(RunSettingsFilePath) --list-tests --nologo -v:q -p:VsTestUseMSBuildOutput=false" ConsoleToMSBuild="true">
```

Makes it a little more understandable/readable.